src/index.html

```diff
@@ -1,15 +1,202 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+                user-scalable=no,
+                initial-scale=1.0,
+                maximum-scale=1.0,minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+    >
+      <fieldset class="box">
+        <legend>
+          Personal information
+        </legend>
+
+        <label class="inside_text" for="surname">
+          Surname:
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            min="5"
+            max="10"
+          >
+        </label>
+
+        <label class="inside_text" for="name">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+          >
+        </label>
+
+        <label class="inside_text" for="age">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="18"
+            max="100"
+          >
+        </label>
+
+        <label class="inside_text" for="date_birth">
```

Comment:
```suggestion
        <label class="inside_text" for="birthDate">
```
Better not to use snake case in class names, because it may be used sometimes as BEM modifyer